app/views/shared/_email_languages.html.erb

```diff
@@ -3,30 +3,33 @@ locals:
 * f: from validated_form_for
 * selection: the current language selection
 %>
-<div class="margin-bottom-4">
-  <ul class="usa-input-list">
-    <% I18n.available_locales.each do |locale| %>
-      <% item_id = "email-locale-#{locale}" %>
-      <li class="grid-col-8 mobile-lg:grid-col-6">
-        <%= f.input_field(
-              :email_language,
-              type: :radio,
-              value: locale,
-              checked: selection ?
-                selection.to_s == locale.to_s :
-                (I18n.locale.to_s == locale.to_s),
-              class: 'usa-radio__input usa-radio__input--bordered',
-              id: item_id,
-            ) %>
-        <%= f.label(
-              :email_language,
-              locale == I18n.locale ?
-                t('account.email_language.default', language: t("i18n.locale.#{locale}")) :
-                t("i18n.locale.#{locale}"),
-              for: item_id,
-              class: 'usa-radio__label width-full',
-            ) %>
-      </li>
-    <% end %>
-  </ul>
+<div class="grid-row margin-bottom-2">
+  <div class="grid-col-fill">
+    <ul class="usa-input-list display-inline-block minw-full">
+      <% I18n.available_locales.each do |locale| %>
+        <% item_id = "email-locale-#{locale}" %>
+        <li class="text-no-wrap width-full">
+          <%= f.input_field(
+                :email_language,
+                type: :radio,
+                value: locale,
+                checked: selection ?
+                  selection.to_s == locale.to_s :
+                  (I18n.locale.to_s == locale.to_s),
+                class: 'usa-radio__input usa-radio__input--bordered',
+                id: item_id,
+              ) %>
+          <%= f.label(
+                :email_language,
+                locale == I18n.locale ?
+                  t('account.email_language.default', language: t("i18n.locale.#{locale}")) :
+                  t("i18n.locale.#{locale}"),
+                for: item_id,
+                class: 'usa-radio__label width-full text-no-wrap',
+              ) %>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+  <div class="grid-col-4 tablet:grid-col-6"></div>
```

Comment:
For posterity: The reason to do it this way where the content column is assigned `grid-col-fill` and this column is given an explicit width is due to how the USWDS grid will assign `flex-grow: 0; flex-shrink: 1` when using explicit widths. This allows the empty column to shrink and allows the `grid-col-fill` to occupy its space.